tests/testing_common.py

```diff
@@ -492,7 +506,14 @@ def _test_merge_layers_nan(self, model_id, config_cls, config_kwargs):
         model = model.to(self.torch_device)
 
         for name, module in model.named_parameters():
-            if "lora_A" in name or "ia3" in name or "lora_E" in name or "lora_B" in name:
+            if (
+                "lora_A" in name
+                or "ia3" in name
+                or "lora_E" in name
+                or "lora_B" in name
+                or "vera_lambda_b" in name
+                or "vera_lambda_d" in name
```

Comment:
```suggestion
                or "vera_lambda" in name
```

would be sufficient, right?